tests/fast/BlobCipherUnitTest.toml

```diff
@@ -1,11 +0,0 @@
-[[knobs]]
```

Comment:
Any reason to delete this? I think we can keep individual test.toml and plus add a consolidated one. For future, we can keep extending the consolidated one, however, still allow looping or running an individual unit test?

Reply:
The consolidated one will run each unit test class once. So for one run of `tests/fast/EncryptionUnitTests.toml` it will run BlobCipherUnitTest and the EncryptKeyProxyUnitTest. Keeping this file seems redundant to me as the consolidated one does the same thing

Reply:
I like the flexibility of running individual tests and looping them rather than requiring to loop them as a single unit. Given the test are lightweight, I can't think of a reason to _not_ leave the current unit test file.
Thanks